src/Compilers/VisualBasic/Portable/Parser/ParseExpression.vb

```diff
@@ -24,14 +36,14 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
 
         ' File: Parser.cpp
         ' Lines: 12312 - 12312
-        ' Expression* .Parser::ParseExpression( [ OperatorPrecedence PendingPrecedence ] [ _Inout_ bool& ErrorInConstruct ] [ bool AllowArrayInitExpression ] )
+        ' Expression* .Parser::ParseExpressionCore( [ OperatorPrecedence PendingPrecedence ] [ _Inout_ bool& ErrorInConstruct ] [ bool AllowArrayInitExpression ] )
```

Comment:
This looks liek a reference to a function in native code. You probably don't want to update this line.